ecmabot.js

```diff
@@ -249,6 +249,16 @@ JSBot.prototype.caniuse = function(context, text) {
 	}
 };
 
-var profile = require("./ecmabot-profile.js");
+var profile;
+if (process.env.ECMABOT_PROFILE) {
+	if (process.env.ECMABOT_PROFILE[0] !== '/') {
+		profile = require(path.join(process.cwd(), process.env.ECMABOT_PROFILE));
```

Comment:
i think we can just use `require(path.resolve(process.env.ECMABOT_PROFILE[0]))` and it will figure all that out for us without the if/else?